README.md

```diff
@@ -43,17 +57,127 @@ Snippets for deployment template scaffolding and 70+ Azure resources.
 | `arm-userfunc-namespace` | Adds a user function namespace to a template. |
 | `arm-` | Displays 70+ snippets for creating Azure resources. For example, type `arm-ubuntu` to add all five resources necessary for a basic Ubuntu virtual machine. |
 
-## Insert Item
+## Azure schema completion and validation
 
-In addition to adding snippets from the code editor, the Insert Item feature can be used to insert new parameters, user-defined functions, variables, resources, and outputs. To do so right-click on the template in the code editor, select **Insert Item** and follow the on-screen prompts.
+The ARM Tools extension uses Azure schemas to provide resource completion and validation. These schemas help you verify that Azure resources are configured correctly before deployment time.
 
-![Inserting an item into an Azure Resource Manager template](./images/insert-item.png)
+### Schema Completions
 
-You can also right-click on any element in the ARM Template Outline view to initiate the insert item process.
+Entering double quotes `""` or selecting `ctrl` + `space` while the cursor is in a resource declaration provides a list of potential properties and property values for each Azure resource.
```

Comment:
This applies to more than list resource declarations

Reply:
Fixed